curriculum/challenges/english/08-coding-interview-prep/rosetta-code/letter-frequency.english.md

```diff
@@ -0,0 +1,82 @@
+---
+id: 5a23c84252665b21eecc7ee1
+title: Letter frequency
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+You are given a string and your task is to calculate frequency of each character.
```

Comment:
Maybe this could read:

Given a string, calculate the frequency of each character.